pricing-page-exercise/saffet/styles.css

```diff
@@ -0,0 +1,257 @@
+body {
+    font-family: sans-serif;
+    margin: 0;
+    padding: 0;
+}
+
+
+/* Navigation bar*/
+.nav-bar{
+    box-sizing: border-box;
+    height: 70px;
+    border: 1px transparent grey;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding-left: 25px;
+    padding-right: 35px;
+    box-shadow: rgba(99, 99, 99, 0.2) 0px 2px 8px 0px;
+}
+
+.nav-bar > p{
+    font-size: 19px; /*company name*/
+}
+
+.nav-links{
+    display: flex;
+    align-items: center;
+}
+
+.nav-links a{
+    text-decoration: none;
+}
+
+.nav-links>a{
+    padding-left: 25px;
+    color: black;
+}
+
+.nav-links>a:hover{
+    text-decoration: underline;
+}
+
+.button{
+    margin-left: 25px;
+    border: 1px solid #007bff;
```

Comment:
I see that you use both standard colors, Hex and rgb/rgba. It's the most common to use HEX (can be combined with rbga if you need to specify opacity). I'd suggest to pic one color type and use it everywhere in the project.

Reply:
Anna some of the colors I generated from the web. Like box shadow for nav so I agreed your advice but just for this example can we accept? I promise, in the next project I will consider :D

Reply:
That's fine, keep them as is for now but going forward please use the same color formats. You can always convert them, there are lots of online tools for that :)